pkg/chartvalues/aws_operator_template.go

```diff
@@ -6,6 +6,11 @@ const awsOperatorTemplate = `Installation:
       Vault:
         Address: http://vault.default.svc.cluster.local:8200
     Guest:
+      IPAM:
+        NetworkCIDR: "10.12.0.0/16"
```

Comment:
Do we always want to use these values? Does it make sense to template them?

Reply:
You mean if the CIDR should be a variable that is then given when template is rendered? I'm not sure. I'm fine to change these to variables, if you think that's a good thing, but for now, I'd prefer less variables as e2e tests are already full of them.

Nevertheless, the actual IPAM test is written so that as long as these values make sense and allow more than four tenant clusters to be created, it doesn't care about specific values so it can be changed in the future without breaking anything.